modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -24,7 +24,7 @@ link:{attachmentsdir}/api.raml[api.raml]
 +
 . Check Add APIkit Components. In the RAML File field, browse to and select the downloaded `api.raml` file.
 +
-image::apikit-tutorial-3e0e4.png[height=633,width=600]
+image::apikit-tutorial-3e0e4.png[height=633,width=600][alt=New Mule Project window for project setting to select .raml file.]
```

Comment:
```suggestion
image::apikit-tutorial-3e0e4.png[height=633,width=600][alt=Project Settings for a New Mule Project, including Project Name, Runtime, Maven Settings, Version Control System Support, APIkit Settings, and API Definition.]
```

Reply:
This is an example of how NOT to use a screenshot. Basically, the screenshot is being used to replace text, which is why the alt text needs to be more detailed. This was probably written by a developer... They love using screenshots instead of writing.